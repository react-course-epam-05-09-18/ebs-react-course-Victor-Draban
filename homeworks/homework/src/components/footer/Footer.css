:local(.footer) {
    position: absolute;
    bottom: 0;
    width: 100%;
    height: 60px;
    background: lavender;
}

:local(.copyright) {
    margin-left: 42px;
    color: rgba(20, 34, 224, 0.61);
}