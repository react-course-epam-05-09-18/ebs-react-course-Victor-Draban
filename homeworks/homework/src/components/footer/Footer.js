import React from 'react';
import styles from "./Footer.css";

var footerContent = <span className = { styles.copyright } >
		    	copyright {new Date().getFullYear()}
		    </span>

export default () => (
	<footer className = { styles.footer }>
		{ footerContent }
	</footer>
);