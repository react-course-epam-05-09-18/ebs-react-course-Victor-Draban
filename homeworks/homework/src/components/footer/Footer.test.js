import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Footer from './Footer';

configure({ adapter: new Adapter() });

describe('>>> Footer --- Shallow Render REACT COMPONENTS', () => {
        it('should render a Footer', () => {
                const wrapper = shallow(
                        <Footer />
                );
                expect(wrapper.length).toEqual(1);
        });
});