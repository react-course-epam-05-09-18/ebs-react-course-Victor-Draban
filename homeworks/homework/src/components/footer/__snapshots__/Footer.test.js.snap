// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> Footer --- Shallow Render REACT COMPONENTS should render a label 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <lable>
    Hello Jest!
</lable>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Hello Jest!",
    },
    "ref": null,
    "rendered": "Hello Jest!",
    "type": "lable",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Hello Jest!",
      },
      "ref": null,
      "rendered": "Hello Jest!",
      "type": "lable",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
